smqtk_iqr/utils/preview_cache.py

```diff
@@ -29,58 +29,49 @@ class PreviewCache (object):
     #   path to the generated preview image.
     PREVIEW_GEN_METHOD: Dict[Hashable, Callable] = {}
 
-    @property
-    def _log(self):
-        return logging.getLogger('.'.join([self.__module__,
-                                           self.__class__.__name__]))
-
-    def __init__(self, cache_dir):
+    def __init__(self, cache_dir: str):
         """
         :param cache_dir: Directory to cache preview image elements into.
-        :type cache_dir: str
         """
         self._cache_dir = os.path.abspath(os.path.expanduser(cache_dir))
         # Cache of preview images for data elements encountered.
-        #: :type: dict[collections.abc.Hashable, str]
-        self._preview_cache = {}
+        self._preview_cache: Dict[Hashable, str] = {}
         self._video_work_dir = os.path.join(cache_dir, 'tmp_video_work')
 
-    def __del__(self):
+    def __del__(self) -> None:
         """
         Cleanup after ourselves.
         """
-        for fp in six.itervalues(self._preview_cache):
-            os.remove(fp)
+        for fp in self._preview_cache.values():
+            os.remove(fp)  # type: ignore
```

Comment:
the `# type: ignore` should be removed here since the `.values()` change causes the `fp` to be correctly typed for use in `os.remove`.

Reply:
Fixed now